package/yast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Mar 11 13:05:14 UTC 2022 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Extend the Package module to force using PackageSystem or
+  PackageAI without having the mode into account (bsc#1196963).
```

Comment:
A bit technical and hard to connect with the real problems for a user checking whether this fixes a problem. Isn't it? This includes much more than extending an internal API. 

I would go for something closer to "AutoYaST clonning (export): fixed the way to check if packages are installed "